pylint_django/checkers/forms.py

```diff
@@ -0,0 +1,62 @@
+"""Models."""
+from astroid.nodes import Assign, ClassDef
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers.utils import check_messages
+from pylint.checkers import BaseChecker
+from pylint.__pkginfo__ import numversion as PYLINT_VERSION
+
+from pylint_django.__pkginfo__ import BASE_ID
+from pylint_django.utils import node_is_subclass
+
+
+MESSAGES = {
+    'W%d04' % BASE_ID: ("Use explicit fields instead of exclude in ModelForm",
+                        'modelform-uses-exclude',
+                        "Replace exclude with fields attribute"),
+}
```

Comment:
This dictionary is used only inside the checker class so move it there. No need to expose extra objects outside.